src/common/updating/installer.cpp

```diff
@@ -192,4 +193,8 @@ namespace updating
         co_return false;
     }
 
+    bool is_old_windows_version()
```

Comment:
I've introduced that function, because using `Is19H1OrHigher` directly from bootstrapper produced obscure unignorable linker warnings caused by internal WinRT structures alignment.

Reply:
viz.:

```log
 warning C4742: '
 struct winrt::impl::factory_cache 
  `struct winrt::impl::factory_cache & __ptr64 __cdecl 
        winrt::impl::get_factory_cache(void)'
          ::`2'::cache'
has different alignment 
```